doc/tools.rst

```diff
@@ -13,8 +13,17 @@ The standard distribution ships with the following tools:
     and obtain useful information like definition of symbols or suggestions for
     completion.
 
-- | `C2nim <https://github.com/nim-lang/c2nim/blob/master/doc/c2nim.rst>`_
```

Comment:
Why was `c2nim` removed from this list?

Reply:
`installation is now separate from nim.`
https://nimble.directory/pkg/c2nim
https://github.com/nim-lang/Nim/issues/11434#issuecomment-500198458
It is a separate package whatsoever, the docs is for tools that come with nim itself.
:thinking:

Reply:
Oh, it seems it should come with nim itself, but does not, because of Bug.
:thinking:

Reply:
c2nim hasn't been shipped with Nim since 0.16.0. Even there, only the Windows package had a binary, posix had nothing.

Need to decide if it should be shipped with Nim or not.

Reply:
@Araq Said [is a Bug on `choosenim`](https://github.com/nim-lang/Nim/issues/11434#issuecomment-500341159), and should be shipped.

The `c2nim` of Schrödinger :thinking: 🤪

Reply:
It is a bug in `koch` and `niminst`. `koch` calls `bundleC2nim()` in `zip` but not in `xz`. So c2nim gets built in `winrelease` since it calls `zip` but it isn't ever done for Linux/OSX builds at all. Secondly, `niminst` doesn't include the `c2nim` binary so even though it got built on Windows, it isn't included in the package.

As for `choosenim`, it simply calls `koch tools` which never calls `bundleC2nim()` so the change really needs to happen in `koch` and `niminst`.

The bigger question is whether we should fix this to include c2nim. It hasn't been in place for years and there haven't been any complaints since it can be installed easily with `nimble install c2nim`.

Reply:
Ok :+1: 
But this PR is about documentation only, 
all the rest kinda out of scope since that poor `tools.rst` aint going to fix Nim code anyways.

:slightly_smiling_face: